### ISSUE TITLE
Double negate counts as a bitmap change

### ISSUE THREAD

**Issue by rj-rl** [bgrabitmap/lazpaint#453]
**Severity**
Minor

**Defect descirption**
Negating an image twice (without any other manipulations) will result in a 'save changes' prompt on closing the file

**Steps to reproduce**
- Open an image
- Select **Colors > Negative** twice (or any odd number of times for that matter)
- Close the file or quit the application

**Expected result**
File (or application) closes quietly, no change is detected

**Actual result**
User is prompted to save changes

**Environment**
Windows 10, build 19042.1415, version 20H2
LazPaint 7.1.6 (64-bit)

https://user-images.githubusercontent.com/84194921/146585458-3fefb0f0-9697-4f31-a9eb-4d711c2f6acc.mp4

**Comment by circular17** [comment]
Yes, a negate (non linear) entails a loss of information, so in theory done twice is not exactly the identity. Though this could be identified as such.
